File: src/main/java/com/example/hnefatafl/model/GameZaloha.java
package com.example.hnefatafl.model;

import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

@Document
public class GameZaloha {
    @Id
    private String id;
    private String type;

    public GameZaloha() {}

    public GameZaloha(String type) {
        this.type = type;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }
}

File: src/main/java/com/example/hnefatafl/model/GameZaloha.kt
package com.example.hnefatafl.model

import org.springframework.data.annotation.Id
import org.springframework.data.mongodb.core.mapping.Document

@Document
class GameZaloha {
    @Id
    var id: String? = null
    var type: String? = null

    constructor()
    constructor(type: String?) {
        this.type = type
    }
}